lib/evss/dependents/service.rb

```diff
@@ -3,34 +3,56 @@
 module EVSS
   module Dependents
     class Service < EVSS::Service
+      include Common::Client::Monitoring
       configuration EVSS::Dependents::Configuration
+      STATSD_KEY_PREFIX = 'api.evss.dependents'
 
       def retrieve
-        perform(:get, 'load/retrieve').body
+        with_monitoring do
+          perform(:get, 'load/retrieve')
```

Comment:
Previously, I had this method returning a `EVSS::Dependents::RetrieveInfoResponse` object.
I refactored it so that it is more like the other methods here, and can be manually called to bypass the cache if needed. 

The only difference is I did not call `body` on the response as I need the full response when caching it in `EVSS::Dependents::RetrievedInfo`